The Intercept.js

```diff
@@ -0,0 +1,243 @@
+{
+	"translatorID": "455fe24a-f7e8-4546-81d2-ad4f9aa10487",
+	"label": "The Intercept",
+	"creator": "czar",
+	"target": "^https?://(www\\.)?theintercept\\.com",
+	"minVersion": "3.0",
+	"maxVersion": "",
+	"priority": 100,
+	"inRepository": true,
+	"translatorType": 4,
+	"browserSupport": "gcsibv",
+	"lastUpdated": "2017-07-09 06:55:10"
+}
+
+/*
+	***** BEGIN LICENSE BLOCK *****
+
+	Copyright © 2017 czar
+	http://en.wikipedia.org/wiki/User_talk:Czar
+
+	This file is part of Zotero.
+
+	Zotero is free software: you can redistribute it and/or modify
+	it under the terms of the GNU Affero General Public License as published by
+	the Free Software Foundation, either version 3 of the License, or
+	(at your option) any later version.
+
+	Zotero is distributed in the hope that it will be useful,
+	but WITHOUT ANY WARRANTY; without even the implied warranty of
+	MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+	GNU Affero General Public License for more details.
+
+	You should have received a copy of the GNU Affero General Public License
+	along with Zotero. If not, see <http://www.gnu.org/licenses/>.
+
+	***** END LICENSE BLOCK *****
+*/
+
+// attr()/text()
+function attr(doc,selector,attr,index){if(index>0){var elem=doc.querySelectorAll(selector).item(index);return elem?elem.getAttribute(attr):null}var elem=doc.querySelector(selector);return elem?elem.getAttribute(attr):null}function text(doc,selector,index){if(index>0){var elem=doc.querySelectorAll(selector).item(index);return elem?elem.textContent:null}var elem=doc.querySelector(selector);return elem?elem.textContent:null}
+
+function detectWeb(doc, url) {
+	if (/theintercept.com\/\d{4}\/\d{2}\/\d{2}\//.test(url)) {
+		return "blogPost";
+	} else if (url.indexOf("/document/") != -1) {
+		return "document";
+	} else if (/(theintercept.com\/search\/\?s=)|(theintercept.com\/?$)/.test(url) && getSearchResults(doc, true) ) {
+		return "multiple";
+	}
+}
+
+function scrape(doc, url) {
+	var item = new Zotero.Item("blogPost");
+	item.blogTitle = "The Intercept";
+	item.language = "en-US";
+	var ldjson = JSON.parse(doc.querySelector('script[type="application/ld+json"]').textContent);
+	item.url = ldjson.url;
+	item.title = ldjson.headline;
+	item.date = ldjson.dateCreated;
+	item.abstractNote = text(doc,'meta[name="description"]');
+	// no item.attachments: Snapshot doesn't work correctly with site
+
+	// Authors
+	if (ldjson.authors.length) {
+		do {
+			item.creators.push(ZU.cleanAuthor(ldjson.authors[0], "author"));
+			ldjson.authors.shift();
+		}
+		while (ldjson.authors.length);
+	}
+
+	// Feature articles json omits the feature title
+	var featureTitle = text(doc,'.Post-feature-title');
+	if (featureTitle){
+		item.title = featureTitle.concat(": ", item.title);
+	}
+	item.complete();
+}
+
+function scrapeDocument(doc, url) {
```

Comment:
The documents have garbage JSON-LD, apart from the time (bad title/url). Thought it was easier to make a separate scrape than jamming it into the main one. I probably could have left out document support altogether, but I had already started.
```JSONLD
{
  "@context": "http://schema.org",
  "@type": "NewsArticle",
  "articleSection": "Document",
  "dateCreated": "2017-02-22T10:59:57+00:00",
  "headline": "Mastering the Internet - The Intercept",
  "thumbnailUrl": "https://www.documentcloud.org/documents/3470275/pages/MTI-p1-thumbnail.gif",
  "url": "https://nodefront.flmcloud.local/document/2017/02/22/mastering-the-internet/"
}
```